x-pack/plugins/ml/public/application/jobs/jobs_list/components/datafeed_modal/datafeed_modal.tsx

```diff
@@ -298,7 +312,43 @@ export const DatafeedModal: FC<DatafeedModalProps> = ({ jobId, end, onClose }) =
                         })}
                         position={Position.Left}
                       />
+                      {showAnnotations ? (
+                        <>
+                          <LineAnnotation
+                            id={i18n.translate(
+                              'xpack.ml.jobsList.datafeedModal.annotationLineSeriesId',
+                              {
+                                defaultMessage: 'Annotations line result',
+                              }
+                            )}
+                            key="annotation-results-line"
+                            domainType={AnnotationDomainType.XDomain}
+                            dataValues={annotationData.line}
+                            marker={<EuiIcon type="annotation" />}
+                            markerPosition={Position.Top}
+                            style={{
+                              line: {
+                                strokeWidth: 3,
+                                stroke: euiTheme.euiColorDangerText,
+                                opacity: 0.5,
+                              },
+                            }}
+                          />
+                          <RectAnnotation
+                            key="annotation-results-rect"
+                            dataValues={annotationData.rect}
+                            id={i18n.translate(
+                              'xpack.ml.jobsList.datafeedModal.annotationRectSeriesId',
+                              {
+                                defaultMessage: 'Annotations rect result',
```

Comment:
Should this `rect` be spelled out 🤔 ?

Reply:
Not seen by user but I do think it would be more clear to spell out 👍